models/users.js

```diff
@@ -178,16 +178,16 @@ const fetchPaginatedUsers = async (query) => {
   }
 };
 
-const fetchAllUsers = async () => {
+const fetchFilteredUsers = async (usernames = []) => {
   try {
     const dbQuery = userModel;
 
-    const snapshot = await dbQuery.get();
+    const snapshot = await dbQuery.where("github_id", "in", usernames).get();
```

Comment:
Is there a limit to how many usernames can exist in the `in` clause?

Reply:
Update: 
![image](https://user-images.githubusercontent.com/68511458/231261409-81505178-6297-4377-ae80-84f00c18780d.png)

Check for an example, `in` supports 30 comparison values.

Thanks for pointing this out, will handle this case.